docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -42,6 +41,8 @@ NOTE: If you don't want to monitor every account in your organization, specify w
 .. Option 2: Temporary keys. To authenticate using temporary keys, refer to the instructions for <<cspm-use-temp-credentials, temporary keys>>.  
 . Once you've selected an authentication method and provided all necessary credentials, click **Save and continue** to finish deployment. Your data should start to appear within a few minutes.
 
+IMPORTANT: Agentless deployment does not work if you are using {cloud}/ec-traffic-filtering-deployment-configuration.html[Traffic filtering].
```

Comment:
Is the "t" supposed to be capitalized here? This doesn't look like an Elastic feature, but I'll defer to you since this is your area of expertise. :)

```suggestion
IMPORTANT: Agentless deployment does not work if you are using {cloud}/ec-traffic-filtering-deployment-configuration.html[traffic filtering].
```